pkg/node/node.go

```diff
@@ -1082,7 +1113,7 @@ func NewBee(ctx context.Context, addr string, publicKey *ecdsa.PublicKey, signer
 		debugService.MustRegisterMetrics(p2ps.Metrics()...)
 		debugService.MustRegisterMetrics(pingPong.Metrics()...)
 		debugService.MustRegisterMetrics(acc.Metrics()...)
-		debugService.MustRegisterMetrics(storer.Metrics()...)
+		// debugService.MustRegisterMetrics(localStore.Metrics()...)
```

Comment:
Why is this commented out? The new localstore metrics should also be registered.